app/views/foreman_tasks/tasks/_details.html.erb

```diff
@@ -79,72 +85,65 @@
   </div>
 </div>
 
-<div>
-  <span class="param-name"><%= _("Id") %>:</span>
-  <span class="param-value"><%= @task.id %></span>
-</div>
-<div>
-  <span class="param-name"><%= _("Label") %>:</span>
-  <span class="param-value"><%= @task.label %></span>
-</div>
-<div>
-  <span class="param-name"><%= _("Name") %>:</span>
-  <span class="param-value"><%= @task.humanized[:action] %></span>
-</div>
-<div>
-  <span class="param-name"><%= _("Owner") %>:</span>
-  <span class="param-value"><%= @task.username %></span>
-</div>
-<div>
-  <span class="param-name"><%= _("Execution type") %>:</span>
-  <span class="param-value"><%= @task.start_at.nil? ? _('Immediate') : _('Delayed') %></span>
-</div>
-<div>
-  <span class="param-name"><%= _("Start at") %>:</span>
-  <span class="param-value"><%= @task.start_at.nil? ? '-' : @task.start_at %></span>
-</div>
-<div>
-  <span class="param-name"><%= _("Start before") %>:</span>
-  <span class="param-value"><%= @task.start_before.nil? ? '-' : @task.start_before %></span>
-</div>
-<div>
-  <span class="param-name"><%= _("Started at") %>:</span>
-  <span class="param-value"><%= @task.started_at.try(:in_time_zone) %></span>
-</div>
-<div>
-  <span class="param-name"><%= _("Ended at") %>:</span>
-  <span class="param-value"><%= @task.ended_at.try(:in_time_zone) %></span>
-</div>
-<div>
-  <span class="param-name"><%= _("State") %>:</span>
-  <span class="param-value"><%= @task.state %></span>
-</div>
-<div>
-  <span class="param-name"><%= _("Result") %>:</span>
-  <span class="param-value"><%= @task.state == 'running' ? '-' : @task.result %></span>
-</div>
-<div>
-  <span class="param-name"><%= _("Params") %>:</span>
-  <span class="param-value"><%= format_task_input(@task) %></span>
-</div>
-<% if @task.parent_task %>
-<div>
-  <span class="param-name"><%= link_to(_("Parent task"), foreman_tasks_task_path(@task.parent_task)) %></span>
-</div>
-<% end %>
-<% if @task.sub_tasks.any? %>
-<div>
-  <span class="param-name"><%= link_to(_("Sub tasks"), sub_tasks_foreman_tasks_task_path(@task)) %></span>
+<div class="row">
+  <div class="col-md-6">
+    <div>
+      <span class="param-name list-group-item-heading"><%= _("Name") %>:</span>
+      <span class="param-value"><%= format_task_input(@task, true) %></span>
+    </div>
+    <div>
+      <span class="param-name list-group-item-heading"><%= _("Result") %>:</span>
+      <span class="param-value">
+        <% if @task.state != 'stopped' %>
+          <%= content_tag(:i, '&nbsp'.html_safe, :class => 'task-status pficon-help') %>
```

Comment:
Is there a way how a user could know a task got paused? If I'm reading this right having a paused task would just show a pretty question mark in the result and make the resume button clickable.